nx-X11/lib/X11/Imakefile

```diff
@@ -1029,19 +1013,8 @@ LCOBJS = \
 	lcUTF8.o
 
 STATICLCOBJS = \
-	$(STATICLCDEPOBJS) \
 	$(STATICI18NSTATICOBJS) 
 
-SHAREDLCOBJS = \
-	$(SHAREDLCDEPOBJS) \
-	$(SHAREDI18NSTATICOBJS) 
-
-OBJS = $(OBJS1) $(OBJS2) $(OBJS3)
-
-EXTRAUNSHAREDOBJS = $(STATICLCOBJS)
```

Comment:
So we're dropping these, because the only remaining libraries are all static and we don't need to build in two different places now? Makes sense, I guess.